src/components/LocationAutocompleteInput/GeocoderGoogleMaps.js

```diff
@@ -45,7 +45,11 @@ class GeocoderGoogleMaps {
    * only relevant for the `getPlaceDetails` function below.
    */
   getPlacePredictions(search) {
-    return getPlacePredictions(search, this.getSessionToken()).then(results => {
+    const limitCountriesMaybe = config.maps.search.limit
+      ? { country: config.maps.search.limit }
+      : {};
+    const restriction = { componentRestrictions: { ...limitCountriesMaybe } };
```

Comment:
Let's not send a key `componentRestrictions` if there is not `countryLimit` set.
```js
 const limitCountriesMaybe = config.maps.search.limit
      ? { 
           componentRestrictions: {
             country: config.maps.search.limit,
           },
         }
      : {};
```
Then `limitCountriesMaybe` could be spread directly to the call:
```js
return getPlacePredictions(search, this.getSessionToken(), limitCountriesMaybe).then(results => {
```